src/bindings/python/src/openvino/properties/properties.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import sys
+from types import BuiltinFunctionType, ModuleType
+from typing import Callable, Any, Union
+
+
+class Property(str):
```

Comment:
Please add some docs here, as it is derivative of a standard string - it might be confusing why it can be called.

Reply:
added